src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v2-monitoring-integration.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Amazon API Gateway V2 Monitoring Integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS API Gateway v2 monitoring integration: what data it reports, and how to enable it."
+freshnessValidatedDate: 2023-07-10
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) now include an integration for AWS API Gateway V2, sending its metrics data to New Relic.
+
+This document explains the integration's features, how to activate it, and what data you can report.
```

Comment:
@rahul188 how about differentiating this from the v1 integration? You could make a pitch here for why users should use this instead of the v1.

Reply:
@rhetoric101, I made the suggested change.

Reply:
By including the purpose of v1 and v2 in the title, it makes this clear from the outset. v1 is for the REST API interface. v2 is for HTTP and Websockets interfaces. Once this is made clear in the title, referring to it as v1 and v2 elsewhere becomes sufficient.